tests/unit/services/users.test.ts

```diff
@@ -0,0 +1,13 @@
+import { getUserDetails, UserDetails } from '../../../src/services/user';
+
+describe('Test User Service', () => {
+  describe('Should process user details and return them', () => {
+    it('should successfully process a jwt token and return the relevant fields', () => {
+      const header = 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCIsImtpZCI6IkFCQ0RFRiJ9';
```

Comment:
I assume from the contents of the test that you generated this token yourself? and it's not a valid token, even if expired

Reply:
Why is it not a valid token?